sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClient.java

```diff
@@ -149,10 +149,14 @@ public class CryptographyClient {
      * @param keyId The Azure Key Vault key identifier to use for cryptography operations.
      * @param pipeline {@link HttpPipeline} that the HTTP requests and responses flow through.
      * @param version {@link CryptographyServiceVersion} of the service to be used when making requests.
+     * @param disableLocalCryptography Indicates if the the ability to perform cryptographic operations locally is
+     * disabled.
      */
-    CryptographyClient(String keyId, HttpPipeline pipeline, CryptographyServiceVersion version) {
+    CryptographyClient(String keyId, HttpPipeline pipeline, CryptographyServiceVersion version,
+                       boolean disableLocalCryptography) {
         this.implClient = new CryptographyClientImpl(keyId, pipeline, version);
         this.keyId = keyId;
+        this.attemptedToInitializeLocalClient = disableLocalCryptography;
```

Comment:
I do something similar in .NET because it's all wired in already, but perhaps you can change the local `attemptedToInitializeLocalClient` to something like `skipInitializingLocalClient`.